README.md

```diff
@@ -50,7 +51,11 @@ logger.info("some interesting log message");
 
 // global context variables can be modified anywhere in the app
 MDC.sessionID = 222;
-logger.error("something has gone wrong", new Error("exception message"));
+logger.error(new Error("exception message"), "something has gone wrong");
```

Comment:
Is there a need to swap the parameters around? If not, we should retain the original.
This also affects my next comment.

```suggestion
logger.error("something has gone wrong", new Error("exception message"));
```

Reply:
@peteriman, I was swapping these around to support the Error setup I have a PR for in the main log4js node package. - i.e. in the main one, I had made it so that in particular the first data entry gets used for location information, though I could adjust that to search through the data for an error object instead like this is currently doing.

Reply:
Oh, do you mean https://github.com/log4js-node/log4js-node/pull/1269?

Reply:
Ah I see... Let's review and merge that PR first before this.

Reply:
I will split this into another PR by itself.